UefiCpuPkg/CpuDxe/CpuPageTable.c

```diff
@@ -1337,7 +1346,15 @@ PageFaultExceptionHandler (
   }
 
   if (NonStopMode) {
-    MpInitLibWhoAmI (&CpuIndex);
+    // MU_CHANGE [START] - CodeQL change
+    Status = MpInitLibWhoAmI (&CpuIndex);
+
+    if (EFI_ERROR (Status)) {
+      DEBUG ((DEBUG_ERROR, "Failed to get processor number.  Failed to get MpInit Processor info.\n"));
+      return;

```

Comment:
Same thought here about being in an exception handler

Reply:
Updated.